content/en/tracing/trace_explorer/trace_queries.md

```diff
@@ -101,11 +101,10 @@ For example, if you query for traces that contain a span from the service `web-s
 {{< img src="tracing/trace_queries/timeseries_using_spans_from.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Timeseries view" >}}
 
 
-## The data that Trace Queries are based on
+## How Trace Queries source data
```

Comment:
Should we update the image and content below to reference diversity sampling as well? Also, we could trim this section and link to the new guide to avoid duplicating too much information.

Reply:
yep, good point, i'll update the screenshot